functions.php

```diff
@@ -119,6 +119,8 @@ function twentynineteen_scripts() {
 
 	wp_enqueue_script( 'twentynineteen-skip-link-focus-fix', get_template_directory_uri() . '/js/skip-link-focus-fix.js', array(), '20151215', true );
 
+	wp_enqueue_style( 'twentynineteen-print-style', get_template_directory_uri() . '/print.css', array(), filemtime(get_template_directory() . '/print.css'), 'print' );
```

Comment:
This looks odd. Is it correct?

Reply:
As mentioned in the docs:

> If version is set to false, a version number is automatically added equal to current installed WordPress version. If set to null, no version is added.

At the moment, main stylesheet has no specific version number supplied and falls back to `4.9.8` (current WordPress version). Is there a need to have a dynamic version number added for `print.css`?

Reply:
I think it was because of my bad monitor, I didn't recognize the function (was seeing n instead of m), and when I suggested a similar way of doing version number for `_s`, it was shot down. I think they said that the file time function was not necessarily available on every server or something like that. I'd have to search for the issue to get the exact reason.

Reply:
> Is there a need to have a dynamic version number added for print.css?

@enodekciw I only added the dynamic version number due to testing purposes to avoid caching issues. 

> I think it was because of my bad monitor, I didn't recognize the function (was seeing n instead of m), and when I suggested a similar way of doing version number for _s, it was shot down. I think they said that the file time function was not necessarily available on every server or something like that. I'd have to search for the issue to get the exact reason.

@joyously As mentioned above, I primarily use `filemtime` to avoid caching issues. However, as `filemtime` is plain PHP I wonder why it should not be available on some servers. 🤔

Reply:
Hey @nielslange, Since it sounds like `filemtime` is only used for caching purposes, I think we can live without it. I think the version number should be synced with the theme’s version number. Remove that when you can and we can go ahead and merge this ;-)

Reply:
@allancole I just updated my PR and replaced `filemtime(get_template_directory() . '/print.css')` with `wp_get_theme()->get( 'Version' )`, to show the themes verion number instead.